tests/qunatizers_infrastructure_tests/pytorch_tests/test_pytorch_base_quantizer.py

```diff
@@ -0,0 +1,133 @@
+# Copyright 2023 Sony Semiconductor Israel, Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+import unittest
+
+from typing import Dict
+
+import torch
+import torch.nn as nn
+
+from model_compression_toolkit.core.common.quantization.node_quantization_config import NodeWeightsQuantizationConfig, \
+    NodeActivationQuantizationConfig
+from model_compression_toolkit import qunatizers_infrastructure as qi, QuantizationConfig
+from model_compression_toolkit.core.common.target_platform import QuantizationMethod
+from model_compression_toolkit.core.tpc_models.default_tpc.latest import get_op_quantization_configs
+from tests.qunatizers_infrastructure_tests.pytorch_tests.base_pytorch_infrastructure_test import \
+    BasePytorchInfrastructureTest
+
+
+class ZeroWeightsQuantizer(qi.BasePytorchQuantizer):
```

Comment:
You decided to no create a "get_quantizer" method in the base infra tests class?
If not, then still this quantizer should be in some common place (in base or some external "infra_tests_quantizers" file), since it's being used across multiple tests and is not specific for this test.

Reply:
you right. I moved it to common file